src/translations/translations_EN.json

```diff
@@ -28,6 +28,7 @@
     "403_page": "You must be logged in for using this feature",
     "404_page": "Got lost? (404-page)",
     "500_page": "Whoops, something went wrong. Try again later",
+    "expired_session": "Whoops, your session has expired. You must log in",
```

Comment:
Foregriper du neste task ? :-)

Burde vi kanskje fjernet "whoops | ups" ?

Reply:
Fjernet expired_session, bruker den ikke.
Liker Whoops | ups, indikerer at problemet ligger hos oss og ikke hos brukeren.

Reply:
veldig mange som har den oops. eller woops, ja. Får bare passe på vi ikke blir for uformelle - slik som f.ex coop obs-appen ;-)